apps/mobile/src/features/send-refactor/utils.ts

```diff
@@ -0,0 +1,125 @@
+import { type BtcFormSchema } from '@/features/send-refactor/forms/btc/btc-form-schema';
+import { type StxFormSchema } from '@/features/send-refactor/forms/stx/stx-form-schema';
+import { type Account } from '@/store/accounts/accounts';
+import { bytesToHex } from '@noble/hashes/utils';
+import { type PublicKey } from '@stacks/common';
+import { ChainId, type StacksNetwork } from '@stacks/network';
+import BigNumber from 'bignumber.js';
+import memoize from 'p-memoize';
```

Comment:
Is this `p-memoize` library truly necessary? It seems like over-engineering to add a new library to memoize validation of address compliance. 

What happens if an address suddenly becomes non-compliant, how will we know if the address is still the same?

Reply:
I think it's related to the `Chainalysis` issue? If so I guess it is needed to stop that 👍

Reply:
This does reduce the requests to Chainalysis, but more importantly avoids a significant UI lag during subsequent validations.